pipeline/pipeline.py

```diff
@@ -175,8 +175,8 @@ async def pipe_strategy(
         PROBLEM_STATEMENT=problem_statement.model_dump_json()
     )
     system_prompt: str = get_strategy_system_template(examples)
-    model: str = "gpt-3.5-turbo-1106"
-    gpt4o: str = "gpt-4o"
+    model: str = "gpt-4o-mini"
+    gpt4o: str = "gpt-4o-mini"
```

Comment:
line 179 was there in case we wanted to up the reasoning capabilities for one of the  following steps, so does not make sense as you currently have it, I suggest that you wither remove this line completely or keep the line as is.